NativeRT readme

dolpm opened the pull request:
Summary: att

Test Plan: ci

Differential Revision: D75557667

yiming0416 commented on torch/nativert/OVERVIEW.md:
```diff
@@ -0,0 +1,415 @@
+
+Values are a internal construct that represent the edges of our graph. In the
+graph, a value could only carry one of these permissible types: Tensor,
+TensorList, SymInt, SymIntList and CustomObg. Values are uniquely identifiable
```
CustomObg -> CustomObj

zhxchen17 reviewed (approved):
Yeah just address the comments from the others.

dolpm committed:
NativeRT readme (#154581)

Summary:

att

Test Plan: ci

Reviewed By: yiming0416, zhxchen17

Differential Revision: D75557667
